src/slash_command.ts

```diff
@@ -65,24 +66,36 @@ export async function handleSlashCommand(
         for (let key in variables) {
             if (key !== 'env_vars') {
                 env_vars[key] = variables[key];
-                allSet &&= await tfcApi.setVariable(workspaceId, existingVars[key], key, variables[key]);
+                allSet &&= await tfcApi.setVariable(existingVars[key], key, variables[key]);
             } else {
                 // do nothing
             }
         }
 
-        let env_vars_string = "{\n";
-        for (let key in env_vars) {
-            env_vars_string += `"${key}"="${env_vars[key]}"\n`
-        }
-        env_vars_string += "}\n";
-        allSet &&= await tfcApi.setVariable(workspaceId, existingVars['env_vars'], 'env_vars', env_vars_string);
```

Comment:
using HCL to store a map in a single variable is unique to TFC. For s3, I'm using the tfvars.json format, so all variable values are represented as JSON. This hcl format function was moved inside of TerraformCloudApi